packages/fxa-auth-client/package.json

```diff
@@ -13,7 +13,6 @@
     "./lib/": "./lib/"
   },
   "scripts": {
-    "lint": "eslint . --ext .ts",
```

Comment:
Q: Why remove the `lint` script from each package?

I see we added it above in nx.json
https://github.com/mozilla/fxa/blob/a23877ae4c03b62f0a3426b4f7cc62b93a5a09ab/nx.json#L17-L19

And seemingly removed from 6 packages/*/package.json files in this PR, but maybe not _all_ package.json files?

```sh
git grep -n '"lint": ' packages/**/package.json | cat

packages/123done/package.json:41:    "lint": "eslint .",
packages/browserid-verifier/package.json:46:    "lint": "eslint .",
packages/fortress/package.json:39:    "lint": "npm-run-all --parallel lint-eslint",
packages/fxa-admin-panel/package.json:13:    "lint": "eslint .",
packages/fxa-admin-server/package.json:11:    "lint": "eslint .",
packages/fxa-auth-server/package.json:19:    "lint": "eslint . .storybook --ext .js,.ts",
packages/fxa-content-server/package.json:13:    "lint": "eslint app server tests --cache",
packages/fxa-customs-server/package.json:21:    "lint": "eslint .",
packages/fxa-geodb/package.json:12:    "lint": "eslint .",
packages/fxa-profile-server/package.json:7:    "lint": "eslint .",
packages/fxa-react/package.json:21:    "lint": "eslint . .storybook",
packages/fxa-shared/package.json:75:    "lint": "eslint .",
```

Reply:
This should all be resolved now.